### DISCUSSION TITLE
please help me with build number issue i tried syncing build and version number too but still error

### DISCUSSION THREAD

**Discussion by omkosolutions** [codemagic-ci-cd/codemagic-docs#2860] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1)i uploaded all the certificates and keys configured it got the API key too for ios test flight uploading ,
2) Build completed successfully.
3)Following submissions will start shortly after App Store Connect completes processing of uploaded artifacts: TestFlight, Beta groups, App Store review.
4)Further logs are available in Post processing section.
5) but post processing fails every time we do it 
down is the error we are getting
Find freshly uploaded build
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
The application binary was successfully uploaded to App Store Connect for processing but the corresponding build is not available in App Store Connect. This usually happens when the uploaded application has one or more issues. Please check your email for direct feedback from Apple.

### Expected results

![problem ss](https://github.com/user-attachments/assets/9334d494-e1bb-4b60-b82d-35049b0da7f5)
the app should have been uploaded hear after our process but did not happen after multiple tries 

### Actual results

it is no uploaded 

### Build id (optional)

com.omkosolutions.halalApp

**Comment by dam-ease** [comment]
Hello @omkosolutions,
From the information shared, this seems to be an issue related to Appstore side of things. Please can you check the email from Apple to see if there are more pointers. Also kindly share the build ID with me? You can find it under your build details page's 'ID' section.

**Reply by omkosolutions** [reply]
ID:
674d995f5be495175959f320
@dam-ease this is the build id please check.

**Reply by dam-ease** [reply]
Thanks for sharing this!
Please can you share the email you received from Apple as well so I can get more insights into this and suggest possible solutions.
